pyxem/signals/diffraction2d.py

```diff
@@ -1008,6 +1012,14 @@ def center_direct_beam(
         electron diffraction pattern and translate it to the center of the
         image square.
 
+        The direct beam position can either be passed to the method with the shifts
+        parameter, or the function can calculate it on its own.
+
+        Note: if the signal has an integer dtype, and subpixel=True is used (the default)
+        the total intensity in the diffraction images will most likely not be preserved.
+        This is due to subpixel=True utilizing interpolation. To keep the total intensity
+        use a float dtype, which can be done by s.change_dtype('float32', rechunk=False).
+
```

Comment:
I think this should be a `Notes` after the parameters, that way we can add "see notes" to the end of the `subpixel` parameter description

Reply:
Done!